packages/acot-preset-wcag/src/rules/focusable-has-indicator.ts

```diff
@@ -0,0 +1,68 @@
+import { createRule } from '@acot/core';
+
+const SELECTOR = [
+  'input',
+  'select',
+  'textarea',
+  'a[href]',
+  'button',
+  '[tabindex]',
+  'audio[controls]',
+  'video[controls]',
+  '[contenteditable]:not([contenteditable="false"])',
+  'details>summary:first-of-type',
+  'details',
+].join(',');
+
+type Options = {};
+
+export default createRule<Options>({
+  type: 'contextual',
+  selector: SELECTOR,
+  immutable: false,
+  meta: {
+    description: 'Focusable element has visible focus indicator',
+    tags: ['wcag21aa', '2.4.7 Focus visible'],
+    recommended: true,
+  },
+
+  test: async (context, node) => {
+    function getComputedOutlineStyleAndCssText(el: Element) {
+      const CSSStyleDeclaration = window.getComputedStyle(el);
+
+      return {
+        outline: {
+          outlineColor: CSSStyleDeclaration.outlineColor,
+          outlineStyle: CSSStyleDeclaration.outlineStyle,
+          outlineWidth: CSSStyleDeclaration.outlineWidth,
+        },
```

Comment:
```suggestion
        outline: {
          color: CSSStyleDeclaration.outlineColor,
          style: CSSStyleDeclaration.outlineStyle,
          width: CSSStyleDeclaration.outlineWidth,
        },
```

I want the name to omit the key because it has duplicate meanings.